src/lib/holocene/modal.svelte

```diff
@@ -66,8 +66,12 @@
     dispatch('confirmModal');
   };
 
+  const closeModal = () => {
+    open = false;
```

Comment:
What did the dispatch('cancelModal') do?

Reply:
In the case of the signal confirmation modal it calls `hideSignalModal` ⬇️ 
https://github.com/temporalio/ui/blob/50a5e10e55c3ea7b0771e58360b2bb994c0615aa/src/lib/components/workflow/client-actions/signal-confirmation-modal.svelte#L70

which calls `resetView()` on the code block
https://github.com/temporalio/ui/blob/50a5e10e55c3ea7b0771e58360b2bb994c0615aa/src/lib/components/workflow/client-actions/signal-confirmation-modal.svelte#L28

Reply:
But like for all modals won't that not be called now?

Reply:
It will still be called because we are updating `open` to `false` which triggers `toggleModal` which then calls `modal?.close()`
https://github.com/temporalio/ui/blob/50a5e10e55c3ea7b0771e58360b2bb994c0615aa/src/lib/holocene/modal.svelte#L44-L52

which then calls `handleCancel`
https://github.com/temporalio/ui/blob/50a5e10e55c3ea7b0771e58360b2bb994c0615aa/src/lib/holocene/modal.svelte#L84

Reply:
🫠 ohhh Modals